pkg/wallet/wallet.go

```diff
@@ -187,11 +191,24 @@ func Imported(nickname string, privateKey string) (*Wallet, error) {
 		return nil, fmt.Errorf("encoding private key B58: %w", err)
 	}
 
-	keypair := ed25519.NewKeyFromSeed(privKeyBytes)
+	wallets, err := LoadAll()
+	if err != nil {
+		return nil, fmt.Errorf("error loadin wallets %w", err)
+	}
 
+	keypair := ed25519.NewKeyFromSeed(privKeyBytes)
 	pubKeyBytes := reflect.ValueOf(keypair.Public()).Bytes() // force conversion to byte array
 
 	addr := blake3.Sum256(pubKeyBytes)
+	//nolint:gomnd
+	Address := "A" + base58.VersionedCheckEncode(addr[:] /* current version is 0*/, 0x00)
```

Comment:
The linter is yelling, I propose to:
- create an explaining variable
- remove the now useless comment

Why are you using an upper case A for the address variable?
```suggestion
	version := 0x00
	address := "A" + base58.VersionedCheckEncode(addr[:], 0x00)
```